.editorconfig

```diff
@@ -9,7 +9,7 @@ root = true
 
 # Change these settings to your own preference
 indent_style = space
-indent_size = 4
+indent_size = 2
 
```

Comment:
Start using 2 space indent. I'll fix it as I go; or I can do all the files in another follow up PR.